scripts/skimNtuple_mib.py

```diff
@@ -294,16 +289,13 @@ def parseInputFileList (fileName) :
         condorFile.write ('Log         = condor_job_$(ProcId).log\n')
         condorFile.write ('Output      = condor_job_$(ProcId).out\n')
         condorFile.write ('Error       = condor_job_$(ProcId).error\n')
-        condorFile.write ('Requirements = ((machine != "hercules.hcms.it")&&(machine != "pcmaster01.hcms.it")&&(machine != "catalina.hcms.it"))\n')
+        condorFile.write ('Requirements = ((machine == "clipper.hcms.it")||(machine == "pccms12.hcms.it"))\n')
```

Comment:
This for instance would be introduced as an additional condition in ```scripts/skimNtuple.py```, with an additional script argument.

Reply:
Thank you for your suggestions !